src/components/modal/modal.scss

```diff
@@ -3,16 +3,18 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-modal-content-padding: Specifies the padding of the modal `content` slot.
- * @prop --calcite-scrim-background: The component's semi-transparent background color.
  * @prop --calcite-modal-content-background: Specifies the background color of content placed in the `content` slot.
+ * @prop --calcite-modal-content-padding: Specifies the padding of the modal `content` slot.
+ * @prop --calcite-modal-scrim-background: Specifies the background color of the modal scrim.
  */
 
 :host {
   @apply absolute flex inset-0 opacity-0 z-overlay;
   visibility: hidden !important;
   transition: visibility 0ms linear var(--calcite-internal-animation-timing-slow),
     opacity var(--calcite-internal-animation-timing-slow) $easing-function;
+  // the modal should always use a dark scrim, regardless of light / dark mode - matches value in global.scss
+  --calcite-modal-scrim-background-internal: #{rgba($blk-240, 0.85)};
```

Comment:
@SkyeSeitz @ashetland - I'm not sure why this was a different value than the regular scrim, I couldn't find an example of this in the Figma kit, but I confirmed it matches scrim. Let me know if this needs to be adjusted.